src/Makefile

```diff
@@ -1,4 +1,4 @@
-PKG_VERSION=0.0.19
```

Comment:
You also need to change `pkg/docker/Dockerfile-xenial` (see #116).

Reply:
Done